### PR TITLE
Add PHP 8.3 support

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -137,8 +139,8 @@ jobs:
             experimental: false
           - operating-system: "ubuntu-20.04"
             php-version: "8.3"
-            dependencies: "highest"
-            experimental: true
```
Comment (mimmi20) [inline .github/workflows/continuous-integration.yml:138]: Please add experimental support for PHP 8.4

**Review by mimmi20 — CHANGES_REQUESTED** [review]
Please add experimental support for PHP 8.4

**Comment by asgrim** [comment]
Hmm not sure why the build hasn't run here, it's not asking to approve the pipeline either?

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (Slamdunk) [reply .github/workflows/continuous-integration.yml:138]: Done

**Comment by Slamdunk** [comment]
Fun fact: the build had a successful run on my fork https://github.com/Slamdunk/browscap-php/actions/runs/6941392168

**Review by mimmi20 — APPROVED** [review]

**Comment by asgrim** [comment]
> Hmm not sure why the build hasn't run here, it's not asking to approve the pipeline either?

@Slamdunk please could you try rebasing, I've just merged #642  which hopefully will fix the build not running

**Review by asgrim — APPROVED** [review]

**Comment by asgrim** [comment]
@Slamdunk thank you - this all green now :)

**Pull request merged**